### PR TITLE
Add wp binary diffing use case

### PR DESCRIPTION
**Pull request by yellowbyte**

### CODE REVIEW

File: wp/plugin/README.md [hunk 1]
```diff
@@ -225,6 +225,53 @@ case NAV:
 And a similar invocation of bap will indeed give `UNSAT`, meaning that
 the return values are always identical for identical inputs.
 
+This plugin can also supplement current binary diffing tools to not 
+only identify code reuse between binaries but also understand the 
+implications and causes of their nuances.
```
Comment (codyroux) [inline wp/plugin/README.md]: I'm not sure "nuances" is the right word here, maybe "nuanced differences in behavior"?

File: wp/plugin/README.md [hunk 2]
```diff
@@ -225,6 +225,53 @@ case NAV:
...
+scores on how closely functions between binaries match as well as how 
+closely the basic blocks within a function match. Those details are 
+helpful to identify what is changed, but most of the time, users want to 
+know more.
+
+For malware attribution and software plagiarism identification, users 
+will want to also know if the detected similarities are authentic. 
+False positive identifications are not uncommon. Multiple functions 
+can have similar control-flow structure and instructions distribution. 
+For authenticity, we have to check the disassembly to see if the
```
Comment (codyroux) [inline wp/plugin/README.md]: "to see if" -> "to see whether"

**Comment by codyroux** [comment]
Looks great! Other than a couple minor suggestions, I'm all for this.

File: wp/plugin/README.md (hunk 1 above)
Comment (yellowbyte) [reply wp/plugin/README.md]: That's true. Just "nuances" is too vague. I won't want to leave room for misinterpretation or misunderstanding. I will change it to "nuanced differences in behavior".

**Comment by yellowbyte** [comment]
Awesome. Just made those changes you suggested. I will merge it now.

**Pull request merged**